### PR TITLE
Put all unit tests in the unit test folder

### PR DESCRIPTION
**Pull request by kd7uiy**

### CODE REVIEW

File: Workshop.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (TomMalbran) [inline Workshop.xml]: You probably shouldn't commit this file. The Workshop test creates this files in the root and never deletes it, which is wrong.

**Comment by dusho** [comment]
Ye. Sorry about that. Was actually using that file as a template for furniture settings in furnitures.xml
For future reference. Is there temp folder used for saving some files? That are maybe in .gitignore?

**Comment by TomMalbran** [comment]
You probably should delete the file after the test. Not sure about a folder, maybe we could have one.

**Comment by kd7uiy** [comment]
There seems to be a /Temp folder, that might be a good place for it?

**Comment by mikejbrown** [comment]
Why is the test reading/writing files at all? It should not. It should be using a string instead.

**Comment by mikejbrown** [comment]
All of the tests are still being found and passing. No unexpected breakages. Merging. The FurnitureWorkshopTest needs to be fixed in its own PR.

**Pull request merged**